Day-II-Calculator/calculator/src/App.css

```diff
@@ -0,0 +1,11 @@
+.app-container {
+  display: flex;
+  flex-direction: column;
+  width: 20%;
```

Comment:
This solution depends on the viewport having a very large width and is always depended on view port width. This causes the layout to be bad on literally any view port other than the one it was developed from.